mozilla-services/consvc-shepherd
# fix campaign summary view

## Problem Statement

The advertiser ID was removed from the campaign summary view, which needs to be fixed.

## Proposed Changes

Added the fix for the advertiser ID in the campaign summary view.

## Verification Steps

Open the campaign summary view in the Django admin to see if the advertiser ID column exists.
